test.log

```diff
@@ -0,0 +1,423 @@
+[INFO] Scanning for projects...
```

Comment:
Ahhh, this was added by mistake. This is the output of "mvn test". I will remove this.